### DISCUSSION TITLE
looking for feedback/review of github profile and few questions

### DISCUSSION THREAD

**Discussion by NirmalSilwal** [EddieHubCommunity/support#233] [Show and tell] [answered]
Hi contributor there,

Can someone provide me genuine feedback on improving my [github profile](https://github.com/NirmalSilwal) ?

@eddiejaoude may be you could do review in next live stream if time permits 🤔 

Also, I have one repo on [Data Structures and Algorithms](https://github.com/NirmalSilwal/Data-Structure-and-Algorithm-Java-interview-kit) . Would like to know whether this kind of repo is also **considered a project** in open source community? Just curious!

![my github profile sept 19 2020](https://user-images.githubusercontent.com/31946761/93662081-cd90e280-fa7a-11ea-99a4-f6e43b32e6b5.png)

Best
Nirmal

**Comment by eddiejaoude** [comment, marked as answer]
I really like your GitHub profile, it is fun but also informational. You should add it to our [awesome-github-profile](https://github.com/EddieJaoudeCommunity/awesome-github-profiles) repo

You have your socials links which is great, but your Instagram is private, so it feels a bit odd having it there - but that is just my personal opinion.

For your project, it looks good, but I think your README needs more love. Also you could add some community love like
- Code of Conduct
- Contributor file
- Issue template
- PR template
etc

I hope that helps.

**Reply by NirmalSilwal** [reply]
I agreed about social links.

I will also be adding my repo to [awesome-github-profile](https://github.com/EddieJaoudeCommunity/awesome-github-profiles) as per your suggestions and contributor file, PR template all.

Thanks for the suggestions.

**Reply by eddiejaoude** [reply]
No problem, Thank you for sharing 🤓
